packages/whats-new/src/whats-new-page.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, GuidePage } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+
+function WhatsNewPage( { description, heading, imageSrc, isLastPage, key, link, pageNumber } ) {
+	const __ = useI18n().__;
+
+	useEffect( () => {
+		recordTracksEvent( 'wpcom_whats_new_slide_view', {
+			slide_number: pageNumber,
+			is_last_slide: isLastPage,
+		} );
+	}, [ isLastPage, pageNumber ] );
+
+	return (
+		<GuidePage className="whats-new-page__container">
+			<div className="whats-new-page__text">
+				{ heading && <h1 className="whats-new-page__heading">{ heading }</h1> }
+				<div className="whats-new-page__description">
+					{ description && <p>{ description }</p> }
+					{ link && (
+						<Button
+							className="whats-new-page__link"
+							href={ link }
+							isTertiary
+							isLink
+							target="_blank"
+						>
+							{ __( 'Learn more', __i18n_text_domain__ ) }
+						</Button>
+					) }
+				</div>
+			</div>
+			<div className="whats-new-page__visual">
+				{ imageSrc && (
+					<img
+						key={ key }
```

Comment:
I'm assuming that using the same key per announcement and not per image is intentional?

Reply:
I went ahead and removed since I'm not using the key for anything in particular!